custom/build.py

```diff
@@ -11,3 +11,9 @@ def finalize_options(self):
 
     def run(self):
         log.info("brrrrrrrrr")
+
+
+def install(dist):
+    # This could be protected by a opt-in condition...
+    build = dist.get_command_obj("build")
+    build.sub_commands = [*build.sub_commands, ("custom-build", None)]
```

Comment:
I decided to create new array instead of extend the existing one, because I think it is cleaner than modifying a class-variable... At least this change would be scoped to the command object.